src/modules/error/__tests__/ContentNotFoundError.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import ContentNotFoundError from '../ContentNotFoundError'
+
+describe('ContentNotFoundError', () => {
+  it('should have correct message', () => {
+    const error = new ContentNotFoundError(
+      { type: 'category', id: '/augsburg/de/test', city: 'augsburg', language: 'language' }
+    )
+
+    expect(error.message).toMatchSnapshot()
+  })
+
+  it('should produce a stacktrace', () => {
+    try {
+      throw new ContentNotFoundError(
+        { type: 'category', id: '/augsburg/de/test', city: 'augsburg', language: 'language' }
+      )
+    } catch (error) {
+      expect(error.stack).toContain('ContentNotFoundError.js:16') /*  This expect depends on the order of the
```

Comment:
Can we remove the line number here?

Reply:
I included it to assert that there is a stack trace. Not sure about an other way.

Reply:
You could use a regexp here and replace the line number with an arbitrary number.
Also the spec file should have a .spec.js ending